client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -0,0 +1,125 @@
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import request from 'superagent';
+import { startsWith, toUpper } from 'lodash';
+import { localize } from 'i18n-calypso';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SearchCard from 'components/search-card';
+import SectionHeader from 'components/section-header';
+import Suggestions from 'components/suggestions';
+import FormButton from 'components/forms/form-button';
+import { getCurrentUser } from 'state/current-user/selectors';
+import { successNotice, errorNotice } from 'state/notices/actions';
+
+const debug = debugFactory( 'calypso:domains:select-ips-tag' );
+
+class SelectIpsTag extends Component {
+	static ipsTagListUrl = 'https://widgets.wp.com/domains/ips-tag-list.min.json';
+
+	state = {
+		ipsTag: '',
+		ipsTagList: [],
+		query: '',
+	};
+
+	componentWillMount() {
+		request.get( SelectIpsTag.ipsTagListUrl ).then( ( response ) => {
+			this.receiveIpsTagList( response.body );
+		} )
+		.catch( ( error ) => {
+			debug( 'Failed to load IPS tag list! ' + error );
+		} );
+	}
+
+	componentDidUpdate() {
+		if ( this.state.query && ( this.state.query !== this.state.ipsTag ) ) {
+			this.hideSuggestions();
+		}
+	}
+
+	receiveIpsTagList = ipsTagList => {
+		this.setState( { ipsTagList } );
+	}
+
+	handleKeyDown = event => this.suggestionsRef.handleKeyEvent( event );
+
+	handleSearchInputChange = query => this.setState( { query, ipsTag: query } );
+
+	handleSuggestionClick = position => {
+		const parsedLabel = position.label.split( ' ' );
+		this.setState( { ipsTag: parsedLabel[ 0 ] } );
+	}
+
+	handleSubmitButtonClick = event => {
+		event.preventDefault();
+	}
+
+	getSuggestions() {
+		return this.state.ipsTagList
+			.filter( hint => this.state.query && startsWith( hint.tag, toUpper( this.state.query ) ) )
+			.map( hint => ( { label: hint.tag + '  (' + hint.registrarName + ')' } ) );
+	}
+
+	setSuggestionsRef = ref => ( this.suggestionsRef = ref );
+
+	hideSuggestions = () => this.setState( { query: '' } );
+
+	render() {
+		const { selectedDomainName, translate } = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Transfer Domain To Another Registrar' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Transfer to another registrar' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/uk-transfer-out)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).